pkg/blockstorage/blockstorage_test.go

```diff
@@ -51,6 +51,9 @@ var _ = Suite(&BlockStorageProviderSuite{storageType: blockstorage.TypeEBS, stor
 var _ = Suite(&BlockStorageProviderSuite{storageType: blockstorage.TypeGPD, storageRegion: "", storageAZ: "us-west1-b"})
 var _ = Suite(&BlockStorageProviderSuite{storageType: blockstorage.TypeGPD, storageRegion: "", storageAZ: "us-west1-c__us-west1-a"})
 
+// Azure takes than 7.5 mins to finish.
+var _ = Suite(&BlockStorageProviderSuite{storageType: blockstorage.TypeAD, storageRegion: "", storageAZ: "westus"})
+
 func (s *BlockStorageProviderSuite) SetUpSuite(c *C) {
 	var err error
 	config := s.getConfig(c, s.storageRegion)
```

Comment:
@s-soo take a look at the s.getConfig() func, you will need to add one more case for Azure and see if the required env variables are set and retrieve them or skip the test.

Reply:
In addition on the above, Please also add a default case to `s.getConfig()` func  that will fail the test if the blockstorage type is not supported.

Reply:
Thanks Supriya.  So without the needed evd variables, all the Azure tests will be skipped.  Is that OK?  I guess we can't have any of our credentials included in the repo.

Reply:
That should be fine. Also please set these required env variables to your test cluster and check if the test passes locally for you.

Reply:
Done.